README.md

```diff
@@ -62,12 +62,26 @@ A fat (shaded) jar is also available:
 
 We require Java (JDK 8 through 15) and Apache Maven (3.2.5 or higher).
 
-```
+```bash
 $ git clone git://github.com/jonathanswenson/starschema-bigquery-jdbc.git
 $ cd starschema-bigquery-jdbc
 $ mvn clean install -DskipTests
 ```
 
+### Local testing
+Testing locally requires access to secret credentials stored in
+[Google Secret Manager](https://console.cloud.google.com/security/secret-manager/secret/bqjdbc-resources-zip/versions?project=346285421363)
+for accessing Looker-owned BigQuery testing tables.
+Once granted access, download and extract the zip file:
+
+```bash
+# Assuming you've already authenticated with `gcloud auth login`:
+$ gcloud config set project secrets-305922
+$ cd src/test/resources
+$ gcloud secrets versions access latest --secret=bqjdbc-resources-zip --format='get(payload.data)' | tr '_-' '/+' | base64 -d > resources.zip
+$ unzip resources.zip
+```
+
```

Comment:
Not sure how we feel about adding this to the README. It's not like the fact that running the tests requires secrets is a secret, and it's not like randoms have access to this cloud project, just feels weird linking to it in a public README.

Reply:
I'm just gonna remove this for now. There's probably a better place to put it in g3docs.